CHANGELOG.md

```diff
@@ -2,15 +2,25 @@
 
 ## 0.6.2 (Unreleased)
 
+BREAKING CHANGES:
+
+* The `nginx_app_protect_remove_config` tag has been changed to `nginx_app_protect_waf_remove_config` to aid in disambiguation.
```

Comment:
Specifically in so far as the config tag goes, we might actually want to make uploading policy files universal to both the WAF and DoS modules (should be easy enough to do if we follow a similar structure as the snippets uploading structure found in the NGINX Config role)